startup/gui/menus.py

```diff
@@ -475,6 +475,14 @@ def __shaderNodeCreator( nodeName, shaderName ) :
 nodeMenu.append( "/Utility/Dot", Gaffer.Dot )
 nodeMenu.append( "/Utility/Switch", functools.partial( Gaffer.SwitchComputeNode, "Switch" ) )
 
+# VDB nodes
+
+import GafferVDB
+import GafferVDBUI
+
+nodeMenu.append( "/VDB/VDB To Mesh", GafferVDB.VDBToMesh, searchText = "VDBToMesh" )
```

Comment:
How about whacking these up above the Utility section, and maybe the Dispatch section too, so they're slightly more prominent?